config/stack/manifests/resources/replicationgroupclass.resource.yaml

```diff
@@ -8,4 +8,4 @@ overview: |
 readme: |
  Cloud-specific resource classes are used to define a reusable configuration for a specific managed service. This class provides a Redis Replication Group, which is satisfied by [Amazon ElastiCache for Redis](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Replication.CreatingRepGroup.html).
 
- Full documentation can be found in the Crossplane [API Reference docs](https://crossplane.io/docs/master/api/crossplaneio/stack-aws/cache-aws-crossplane-io-v1alpha2.html#ReplicationGroupClass).
\ No newline at end of file
+ Full documentation can be found in the Crossplane [API Reference docs](https://crossplane.io/docs/master/api/crossplaneio/stack-aws/cache-aws-crossplane-io-v1alpha3.html#ReplicationGroupClass).
```

Comment:
same here, isn't this `v1beta1` now? https://github.com/crossplaneio/stack-aws/blob/master/apis/cache/v1beta1/replication_group_types.go#L578